orchestra/admin.py

```diff
@@ -0,0 +1,190 @@
+        return obj.project.workflow_version
+
+    def edit_project(self, obj):
+        return edit_link(obj.project, obj.project.short_description)
+
+
+@admin.register(TaskAssignment)
+class TaskAssignmentAdmin(admin.ModelAdmin):
+    list_display = (
+        'id', 'edit_project', 'edit_task', 'assignment_counter', 'edit_worker',
+        'workflow_version', 'start_datetime')
+    ordering = ('-task__project', 'task__start_datetime', 'assignment_counter')
+    search_fields = (
+        'task__project__short_description', 'task__step__name',
+        'worker__user__username')
+    list_filter = ('task__step__is_human', 'task__project__workflow_version')
+
+    def workflow_version(self, obj):
+        return obj.task.project.workflow_version
+
+    def edit_task(self, obj):
+        return edit_link(obj.task, obj.task.step.name)
+
+    def edit_project(self, obj):
+        return edit_link(obj.task.project, obj.task.project.short_description)
+
+    def edit_worker(self, obj):
+        return edit_link(obj.worker)
+
+
+@admin.register(TimeEntry)
+class TimeEntryAdmin(admin.ModelAdmin):
+    list_display = ('id', 'date', 'worker', 'time_worked', 'assignment')
+    search_fields = (
+        'id', 'worker__user__username', 'assignment__task__step__name',
+        'assignment__task__project__short_description')
+    list_filter = ('worker',)
+
+
+@admin.register(Worker)
+class WorkerAdmin(admin.ModelAdmin):
+    list_display = ('id', 'edit_user', 'email', 'slack_username',)
+    ordering = ('user__username',)
+    search_fields = ('user__username', 'user__email', 'slack_username')
+
+    def edit_user(self, obj):
+        return edit_link(obj.user, obj.user.username)
+
+    def email(self, obj):
+        return obj.user.email
+
+
+@admin.register(WorkerCertification)
+class WorkerCertificationAdmin(admin.ModelAdmin):
+    list_display = ('id', 'worker', 'certification', 'role', 'task_class')
+    search_fields = (
+        'worker__user__username', 'certification__slug', 'certification__name',
+        'certification__workflow__slug', 'certification__workflow__name',
+        'certification__workflow__description')
+    ordering = ('-created_at',)
+    list_filter = ('task_class', 'role')
+
+
+@admin.register(Workflow)
+class WorkflowAdmin(admin.ModelAdmin):
+    list_display = (
+        'id', 'slug', 'name', 'description')
+    ordering = ('slug',)
+    search_fields = ('slug', 'name', 'description',)
+
+
+@admin.register(WorkflowVersion)
+class WorkflowVersionAdmin(admin.ModelAdmin):
+    list_display = (
+        'id', 'slug', 'workflow', 'name', 'description')
+    ordering = ('workflow__slug',)
+    search_fields = ('workflow__slug', 'slug', 'name', 'description',)
+    list_filter = ('workflow',)
+
+
+def edit_link(instance, text=None):
+    if not instance:
+        return None
+    if text is None:
+        text = str(instance)
+    change_url_name = 'admin:{}_change'.format(instance._meta.db_table)
```

Comment:
sanity check---the views are named after the DB tables?  it feels like that breaks encapsulation, but if you've read that this is how it works, then OK